test/NaCl.Core.SimdTests/Salsa20ScalarTests.cs

```diff
@@ -0,0 +1,432 @@
+﻿namespace NaCl.Core.SimdTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Net.Http;
+    using System.Security.Cryptography;
+    using System.Text;
+
+    using FluentAssertions;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    using Base;
+    using Internal;
+    using Vectors;
+    using System.Linq;
+    using NaCl.Core.Base.SalsaCore;
+    using NaCl.Core.SimdTests.Vectors;
```

Comment:
I don't think this `using` is needed since above it you are already specifying `using Vectors;`